### ISSUE TITLE
Vue and NPM 14 Circular dependency due to dependency on buggy shelljs

### ISSUE THREAD

**Issue by mcshaz** [storybookjs/storybook#11173]
**Describe the bug**
the current version of storybook/vue has the following dependency tree:
```
`-- @storybook/vue@5.3.19
  `-- @storybook/core@5.3.19
    `-- shelljs@0.8.3
```
The issue with Node 14 and this version of shelljs is described at https://github.com/shelljs/shelljs/issues/991 - the error is described perfectly in the linked bug report  (resolved).

As an aside to my current issue, it looks like storybook CLI (https://github.com/storybookjs/storybook/blob/next/lib/cli/package.json) has the same dependency.

Could this dependency please be bumped at least 1 minor version, and then released on NPM - thank you.

**Comment by shilman** [comment]
cc @ndelangen

**Comment by ndelangen** [comment]
Is it just a matter of upgrading?

**Comment by mcshaz** [comment]
> Is it just a matter of upgrading?

Yes. All the information I have is from the issue on the shelljs github site (linked). The 1 minor version change from 0.8.3 to 0.8.4 

taken from #https://github.com/shelljs/shelljs/issues/991#issuecomment-619316520
> This should be released as v0.8.4 :shipit: tada This only differs from v0.8.3 by pulling in the referenced change.

The single line dif is at https://github.com/shelljs/shelljs/commit/05374a749992969bb6b853095f4c9308df69aa82

**Comment by ndelangen** [comment]
@mcshaz would you be open to opening a PR to fix this?
